Project.toml

```diff
@@ -4,18 +4,20 @@ keywords = ["multivariate statistics", "dimensionality reduction"]
 license = "MIT"
 desc = "A Julia package for multivariate statistics and data analysis"
 repository = "https://github.com/JuliaStats/MultivariateStats.jl.git"
-version = "0.9.0"
+version = "0.9.1"
 
 [deps]
 Arpack = "7d9fca2a-8960-54d3-9f78-7d1dccf2cb97"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsAPI = "82ae8749-77ed-4fe6-ae5f-f523153014b0"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 Arpack = "0.3, 0.4, 0.5"
-StatsBase = "0.29, 0.30, 0.31, 0.32, 0.33"
+StatsBase = "~0.33"
+StatsAPI = "~1.2"
```

Comment:
Why did you use tilde here? For StatsBase it's equivalent to just `"0.33"`, but for StatsAPI this makes MultivariateStats incompatible with StatsAPI 1.3, which is going to create massive conflicts once some packages start requiring it (e.g. because it includes a new function definition they need).

Reply:
You are correct. I should have loosen bounds.